build.sh

```diff
@@ -13,7 +13,7 @@ cd build
 
 npm install --silent
 
-./node_modules/.bin/pkg . --targets node12-win-x64 --output ../bin/bgpalerter-win-x64
+#./node_modules/.bin/pkg . --targets node12-win-x64 --output ../bin/bgpalerter-win-x64
```

Comment:
I think this can be removed, we only use linux

Reply:
But if we intend to contribute back...

Reply:
a bit of an optimization, but keeping it for now if we ever want to push this back upstream

Reply:
actual michael comment to keep, so I uncommented it and it would build all platforms